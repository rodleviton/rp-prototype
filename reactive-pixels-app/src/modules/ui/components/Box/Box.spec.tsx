import * as React from 'react';
import { render } from 'react-testing-library';
import WithTheme from '../../utils/test/WithTheme';
import { Box } from './';

describe('<Box />', () => {
  it('should render <Box /> component', () => {
    const { container } = render(
      <WithTheme>
        <Box />
      </WithTheme>,
    );

    expect(container.firstChild).toMatchSnapshot();
  });
});
