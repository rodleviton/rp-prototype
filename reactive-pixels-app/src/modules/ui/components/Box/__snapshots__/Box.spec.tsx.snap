// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> should render <Box /> component 1`] = `
<div
  class="css-0 sc-bdVaJa bDWFJH"
/>
`;
